libs/react-components/src/tedi/components/cards/card/card-stories-templates.tsx

```diff
@@ -0,0 +1,546 @@
+          </CardContent>
+        </Card>
+      </Col>
+    </Row>
+    <Row>
+      <Col>
+        <Card>
+          <CardContent>
+            <Text modifiers="bold">Title</Text>
+            <Text color="secondary">Description</Text>
+          </CardContent>
+        </Card>
+      </Col>
+    </Row>
+    <Row>
+      <Col>
+        <Card>
+          <CardContent>
+            <Text modifiers="bold">Title</Text>
+            <Row>
+              <Col>
+                <Text color="secondary">Description</Text>
+              </Col>
+              <Col width="auto">
+                <StatusBadge color="brand">Approved</StatusBadge>
+              </Col>
+            </Row>
+          </CardContent>
+        </Card>
+      </Col>
+    </Row>
+    <Row>
+      <Col>
+        <Card>
+          <CardContent>
+            <Row>
+              <Col className="display-flex align-items-center gap-3">
+                <Icon name="monitor_heart" />
+                <Text color="secondary">Description</Text>
+              </Col>
+            </Row>
+          </CardContent>
+        </Card>
+      </Col>
+    </Row>
+    <Row>
+      <Col>
+        <Card>
+          <CardContent>
+            <Row>
+              <Col className="display-flex align-items-center gap-3">
+                <Icon name="monitor_heart" />
+                <VerticalSpacing size={0}>
+                  <Text modifiers="bold">Title</Text>
+                  <Text color="secondary">Description</Text>
+                </VerticalSpacing>
+              </Col>
+            </Row>
+          </CardContent>
+        </Card>
+      </Col>
+    </Row>
+    <Row>
+      <Col>
+        <Card>
+          <CardContent>
+            <Row>
+              <Col className="display-flex align-items-center gap-3">
+                <Icon name="monitor_heart" />
+                <VerticalSpacing size={0}>
+                  <Text modifiers="bold">Title</Text>
+                  <Text color="secondary">Description</Text>
+                </VerticalSpacing>
+              </Col>
+              <Col width="auto" className="display-flex align-items-center">
+                <Button>Create</Button>
+              </Col>
+            </Row>
+          </CardContent>
+        </Card>
+      </Col>
+    </Row>
+    <Row>
+      <Col lg={6} sm={12}>
+        <Card>
+          <CardContent>
+            <Row>
+              <Col>
+                <Text modifiers="bold">Title</Text>
+                <Text color="secondary">Description</Text>
+                <Separator spacing={1.5} />
+                <Row justifyContent="center">
+                  <Col width="auto">
+                    <Button>Create</Button>
+                  </Col>
+                </Row>
+              </Col>
+            </Row>
+          </CardContent>
+        </Card>
+      </Col>
+    </Row>
+  </VerticalSpacing>
+);
+
+export const CardInfoTemplate: StoryFn<CardStory> = () => (
+  <VerticalSpacing>
+    <Row>
+      <Col>
```

Comment:
This Row/Col combo is not needed, VerticalSpacing already gives space between Cards.